tests/features/project.feature

```diff
@@ -8,9 +8,13 @@ Feature: Project content creation
   Scenario: Creation of a Project content through the UI.
     Given I am logged in as a user with the "create oe_project content, access content, edit own oe_project content, view published skos concept entities, manage corporate content entities" permission
     And the following images:
-      | name    | file           | alt                |
-      | Image 1 | example_1.jpeg | Alternative text 1 |
-      | Image 2 | example_1.jpeg | Alternative text 2 |
+      | name    | file            | alt                 |
+      | Image 1 | example_1.jpeg  | Alternative text 1  |
+      | Image 2 | example_1.jpeg  | Alternative text 2  |
+      | Image 3 | example_1.jpeg  | Alternative text 3  |
+    And the following Remote video:
+      | url                                         | title                   |
```

Comment:
Let's make rule definition consistent with "Given the following image(s)":
```
| name | url |
```

Reply:
the (s) is like a wildcard when you can use both video and videos, since we use 1 video, we formulate as singular

Reply:
I mean to use name as first value and url as second.

Reply:
we actually dont need the name for remote video, I forgot to delete that, the name is extracted out of the video source